temporalcli/commands.operator_search_attribute.go

```diff
@@ -0,0 +1,167 @@
+		return enums.INDEXED_VALUE_TYPE_DOUBLE, nil
+	case "bool":
+		return enums.INDEXED_VALUE_TYPE_BOOL, nil
+	case "datetime":
+		return enums.INDEXED_VALUE_TYPE_DATETIME, nil
+	case "keywordlist":
+		return enums.INDEXED_VALUE_TYPE_KEYWORD_LIST, nil
+	default:
+		return enums.INDEXED_VALUE_TYPE_UNSPECIFIED, fmt.Errorf("unsupported search attribute type: %v, supported types are: [Text, Keyword, Int, Double, Bool, Datetime, KeywordList]", search)
+	}
+}
+
+func (c *TemporalOperatorSearchAttributeRemoveCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	yes, err := cctx.promptYes(
+		fmt.Sprintf("You are about to remove search attributes %v? y/N", c.Name), c.Yes)
+	if err != nil {
+		return err
+	} else if !yes {
+		// We consider this a command failure
+		return fmt.Errorf("user denied confirmation, operation aborted")
+	}
+
+	names := c.Name
+	namespace := c.Parent.Parent.Namespace
+	if err != nil {
+		return err
+	}
+
+	request := &operatorservice.RemoveSearchAttributesRequest{
+		SearchAttributes: names,
+		Namespace:        namespace,
+	}
+
+	_, err = cl.OperatorService().RemoveSearchAttributes(cctx, request)
+	if err != nil {
+		return fmt.Errorf("unable to remove search attributes: %w", err)
+	}
+
+	// response contains nothing
+	return nil
+}
+
+func (c *TemporalOperatorSearchAttributeListCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	request := &operatorservice.ListSearchAttributesRequest{
+		Namespace: c.Parent.Parent.Namespace,
+	}
+	resp, err := cl.OperatorService().ListSearchAttributes(cctx, request)
+	if err != nil {
+		return fmt.Errorf("unable to list search attributes: %w", err)
+	}
+	if cctx.JSONOutput {
+		return cctx.Printer.PrintStructured(resp, printer.StructuredOptions{})
+	}
+
+	type saNameType struct {
+		Name string `json:"name"`
+		Type string `json:"type"`
+	}
+	type saStorageSchema struct {
+		Name   string `json:"name"`
+		Schema string `json:"schema"`
+	}
+
+	var sas []saNameType
+	for saName, saType := range resp.SystemAttributes {
+		sas = append(sas, saNameType{
+			Name: saName,
+			Type: saType.String(),
+		})
+	}
+	for saName, saType := range resp.CustomAttributes {
+		sas = append(sas, saNameType{
+			Name: saName,
+			Type: saType.String(),
+		})
+	}
+	var saSchemaMapping []saStorageSchema
+	for saName, saSchema := range resp.StorageSchema {
+		saSchemaMapping = append(saSchemaMapping, saStorageSchema{Name: saName, Schema: saSchema})
+	}
+
+	cctx.Printer.PrintStructured(sas, printer.StructuredOptions{Table: &printer.TableOptions{}})
+	if len(saSchemaMapping) > 0 {
+		cctx.Printer.Println(color.MagentaString("Storage Schema Mapping:"))
+		return cctx.Printer.PrintStructured(saSchemaMapping, printer.StructuredOptions{Table: &printer.TableOptions{}})
+	}
```

Comment:
Is this a new feature we're adding to CLI as part of this PR?

Reply:
Didn't realize this was not exposed in existing CLI. Remove it for now.